pkg/bundle/installer.go

```diff
@@ -73,13 +73,22 @@ func doInstall(client api.Client, recipe *api.Recipe, repo, name string, refresh
 		return err
 	}
 
+	posthogProperty := utils.PosthogProperties{
+		ApplicationName: repo,
+		RecipeName:      recipe.Name,
+	}
 	if err := performTests(context, recipe); err != nil {
+		posthogProperty.Error = fmt.Errorf("failed to perform tests")
+		utils.PosthogCapture(utils.InstallPosthogEvent, posthogProperty)
 		return err
 	}
 
 	if err := client.InstallRecipe(recipe.Id); err != nil {
+		posthogProperty.Error = fmt.Errorf("failed install recipe")
```

Comment:
should we just add this to our existing error logging wrapper?

Reply:
done

Reply:
I have moved error handling to `tracked` validator